components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -63,11 +63,24 @@ export const PrebuildDetailPage: FC = () => {
     const { toast, dismissToast } = useToast();
     const [currentPrebuild, setCurrentPrebuild] = useState<Prebuild | undefined>();
     const [logNotFound, setLogNotFound] = useState(false);
+    const [noTasksPresent, setNoTasksPresent] = useState(false);
     const [selectedTaskId, actuallySetSelectedTaskId] = useState<string | undefined>(
         window.location.hash.slice(1) || undefined,
     );
 
-    const taskId = selectedTaskId ?? currentPrebuild?.status?.taskLogs.filter((f) => f.logUrl)[0]?.taskId ?? "0";
+    const isImageBuild =
+        currentPrebuild?.status?.phase?.name === PrebuildPhase_Phase.QUEUED &&
+        !!currentPrebuild.status.imageBuildLogUrl;
+    const taskId = useMemo(() => {
+        if (!currentPrebuild) {
+            return null;
+        }
+        if (isImageBuild) {
```

Comment:
This is important and forces the current view to be image build when the image is building. No other tasks are accessible or clickable at that time.
<img width="512" alt="image" src="https://github.com/gitpod-io/gitpod/assets/29888641/1dd865e2-cc40-40c9-acf9-717b4419d18c">